trainings/sourcegraph-101/lesson-plan.md

```diff
@@ -154,7 +179,33 @@ With our query filters, you can narrow things down to find just the content you
 - [Three ways to search code with Sourcegraph](https://learn.sourcegraph.com/three-ways-to-search-code-with-sourcegraph)
 - [Sourcegraph code search cheat sheet](https://learn.sourcegraph.com/how-to-search-code-with-sourcegraph-a-cheat-sheet)
 
-## Unit 4: Code intelligence
+## Unit 4: Code Monitoring
+
+**Learning goals:** By the end of this unit, users will now how to configure a code monitor.
+
+### Configuring a code monitor
+
+Previously, you saw me run a `type:diff` search, in order to see changes to the code. If I want to be proactively informed about that search via email, Slack, or webhooks, that's possible too! To do so, I'll want to set up a code monitor.
+
+🔎 The trainer should show:
+
+* The example use cases on the "getting started" code monitor page
+* Creating a new code monitor using "create copy of monitor" and applying a `repo:` filter
+* The fact that monitors can be connected to email, Slack, or webhooks
+
+❗️Typically, actually going through the entire Slack flow isn't necessary; instead, you can simply speak to it and share a recording afterwards if folks want more info. 
+
+❗️If the customer is interested in email monitors, identify a point person to configure the SMTP connection on the instance; without that, it won't be possible to use them.
```

Comment:
> it won't be possible to use them

nit: Slack and webhook monitors are usable without SMTP

Reply:
Yep! That's why it says "email monitors". :) Is there preferred phrasing?

Reply:
oh lol. 🤦 

Nope. Just ignore me.

Reply:
Hahah, all good—thank you for taking a look, super appreciate it! :)